cernopendata_client/cli.py

```diff
@@ -96,7 +96,7 @@ def get_metadata(server, recid, doi, title, output_fields):
             )
             sys.exit(1)
     else:
-        output_json = record_json
+        output_json = record_json["metadata"]
```

Comment:
Works fine :+1: We could also delete `_files.bucket` and `_files.version_id` because people cannot use them...

```
  "_files": [
        {
            "bucket": "61319316-0704-4b6d-be7c-14017099015c",
            "checksum": "adler32:ff63668a",
            "key": "BuildFile.xml",
            "size": 305,
            "version_id": "713b7316-a48a-4a20-981d-7836740f4f8b"
        },
```